solidity/contracts/utility/TokenHolder.sol

```diff
@@ -33,11 +33,10 @@ contract TokenHolder is ITokenHolder, Owned, Utils {
         IERC20 _token,
         address _to,
         uint256 _amount
-    ) public virtual override ownerOnly validAddress(address(_token)) validAddress(_to) notThis(_to) {
+    ) public virtual override ownerOnly validAddress(address(_token)) validAddress(_to) {
```

Comment:
Why bother checking `_token` when the function will revert anyway if it doesn't point to a valid `IERC20` instance?

Reply:
I agree. @yudilevi, what do you think?

Reply:
sure, we can remove it. but on the other hand, can probably use `validExternalAddress` instead of `validAddress`

Reply:
This is what I've already done in the next PR. I didn't want to keep the `notThis` check because it's recoverable

Reply:
cool